### PR TITLE
Add a recipe for celiagg

### PR DESCRIPTION
**Pull request by jwiggins**

### CODE REVIEW

**Comment by jwiggins** [comment]
Barring any rate-limit CI failures, this should be good to go.

**Comment by scopatz** [comment]
Seems like there are some failures @jwiggins

**Comment by jwiggins** [comment]
On Travis and CircleCI:
```
Traceback (most recent call last):
  File "setup.py", line 82, in <module>
    packages=['celiagg', 'celiagg.tests']
  File "/Users/travis/miniconda/envs/_build_placehold_placehold_placehold_placehold_plac/lib/python3.4/site-packages/numpy/distutils/core.py", line 135, in setup
    config = configuration()
  File "setup.py", line 51, in configuration
    config.add_subpackage('celiagg')
  File "/Users/travis/miniconda/envs/_build_placehold_placehold_placehold_placehold_plac/lib/python3.4/site-packages/numpy/distutils/misc_util.py", line 1000, in add_subpackage
    caller_level = 2)
  File "/Users/travis/miniconda/envs/_build_placehold_placehold_placehold_placehold_plac/lib/python3.4/site-packages/numpy/distutils/misc_util.py", line 969, in get_subpackage
    caller_level = caller_level + 1)
  File "/Users/travis/miniconda/envs/_build_placehold_placehold_placehold_placehold_plac/lib/python3.4/site-packages/numpy/distutils/misc_util.py", line 882, in _get_configuration_from_setup_py
    ('.py', 'U', 1))
  File "/Users/travis/miniconda/envs/_build_placehold_placehold_placehold_placehold_plac/lib/python3.4/site-packages/numpy/compat/py3k.py", line 112, in npy_load_module
    return importlib.machinery.SourceFileLoader(name, fn).load_module()
AttributeError: 'module' object has no attribute 'machinery'

Command failed: /bin/bash -x -e /Users/travis/miniconda/conda-bld/work/celiagg-0.1.1/conda_build.sh
```

I don't really know what to make of that. Should I just nix 3.4 support? Is something else going on unrelated to me? Should I _not_ put `numpy x.x` in my recipe?

**Comment by scopatz** [comment]
I have not seen this error before, but since you are compiling against numpy, you need `numpy x.x`.  If it works without v3.4 support, you can nix it for the time being and fix it in the feedstock later if desired.

**Comment by jwiggins** [comment]
Hmm.. What's the best practice for disabling specific versions of Python?

File: recipes/celiagg/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - freetype 2.6.*
+
+test:
+  imports:
+    - celiagg
+
+about:
+  home: https://github.com/celiagg/celiagg
+  license: MIT
+  summary: 'Anti-Grain Geometry 2.4 for Python with Cython'
```
Comment (scopatz) [inline recipes/celiagg/meta.yaml]: This is the last piece I'd need.

File: recipes/celiagg/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  url: https://github.com/celiagg/celiagg/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win or py34]
+  script: python setup.py install
+
+requirements:
+  build:
```
Comment (jakirkham) [inline recipes/celiagg/meta.yaml]: Also could you please add `toolchain`.

File: recipes/celiagg/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{% set version = "0.1.1" %}
+{% set sha256 = "13ff68c46e7a798bdd2af9f6a1c027dc9cac01e01e3845e11309ab02ecc32243" %}
+
+package:
+  name: celiagg
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
```
Comment (jakirkham) [inline recipes/celiagg/meta.yaml]: Would suggest changing this to `celiagg-{{ version }}.tar.gz`.

Comment (jwiggins) [inline recipes/celiagg/meta.yaml]: Done. I couldn't find the documentation for this part of the `source` section, but I'm hoping this is a local rename that conda performs?

File: recipes/celiagg/meta.yaml (hunk 1 above)
Comment (jwiggins) [inline recipes/celiagg/meta.yaml]: Done

**Comment by jwiggins** [comment]
Okie Dokie. That should have it now.

**Comment by scopatz** [comment]
Thanks @jwiggins!

**Comment by jwiggins** [comment]
Thanks for reviewing!

**Pull request merged**